benchmark-scripts/README.md

```diff
@@ -1,80 +1,3 @@
 # Benchmarking
```

Comment:
We can remove this entire file now that it is obsolete.

Reply:
ok